src/5e-SRD-Monsters.json

```diff
@@ -10234,65 +10593,87 @@
-              },
-              {
-                "name": "Horns",
-                "notes": "When Fire Breath is available",
-                "count": 1,
-                "type": "melee"
-              },
-              {
-                "name": "Claws",
-                "notes": "When Fire Breath is available",
-                "count": 1,
-                "type": "melee"
-              }
-            ],
-            [
-              {
-                "name": "Bite",
-                "notes": "When Fire Breath is available",
-                "count": 1,
-                "type": "melee"
+          "type": "action",
+          "from": {
+            "option_set_type": "options_array",
+            "options": [
+              {
+                "option_type": "multiple",
+                "items": [
+                  {
+                    "option_type": "action",
+                    "action_name": "Bite",
+                    "count": 1,
+                    "type": "melee"
+                  },
+                  {
+                    "option_type": "action",
+                    "action_name": "Horns",
+                    "count": 1,
+                    "type": "melee"
+                  },
+                  {
+                    "option_type": "action",
+                    "action_name": "Claws",
+                    "count": 1,
+                    "type": "melee"
+                  }
+                ]
               },
               {
-                "name": "Fire Breath",
-                "notes": "When Fire Breath is available",
-                "count": 1,
-                "type": "ability"
+                "option_type": "multiple",
+                "items": [
+                  {
+                    "option_type": "action",
+                    "action_name": "Fire Breath",
+                    "notes": "When Fire Breath is available",
+                    "count": 1,
+                    "type": "ability"
+                  },
+                  {
+                    "option_type": "action",
+                    "action_name": "Horns",
+                    "notes": "When Fire Breath is available",
+                    "count": 1,
+                    "type": "melee"
+                  },
+                  {
+                    "option_type": "action",
+                    "action_name": "Claws",
+                    "notes": "When Fire Breath is available",
+                    "count": 1,
+                    "type": "melee"
+                  }
+                ]
               },
               {
-                "name": "Claws",
-                "notes": "When Fire Breath is available",
-                "count": 1,
-                "type": "melee"
+                "option_type": "multiple",
+                "items": [
+                  {
+                    "option_type": "action",
+                    "action_name": "Bite",
+                    "notes": "When Fire Breath is available",
```

Comment:
I wonder if there should be a `notes` or `desc` on the overarching choice instead of repeated in all 3 items.

Reply:
The `desc` of the multiattack action itself should be good enough to indicate which of the options can be picked, so the `notes` field seems a bit redundant, especially on each individual `action` Options. If we feel that more than the top-level Action's `desc` field is needed, perhaps a `desc` can be added to each of the `multiple` Options, like so:

```json
{
  "name": "Multiattack",
	"desc": "The chimera makes three attacks: one with its bite, one with its horns, and one with its claws. When its fire breath is available, it can use the breath in place of its bite or horns.",
  "options": {
    "choose": 1,
    "type": "action",
    "from": {
      "option_set_type": "options_array",
      "options": [
        {
          "desc": "The chimera makes three attacks: one with its bite, one with its horns, and one with its claws."
          "option_type": "multiple",
          "items": [
            {
              "option_type": "action",
              "action_name": "Bite",
              "count": 1,
              "type": "melee"
            },
            {
              "option_type": "action",
              "action_name": "Horns",
              "count": 1,
              "type": "melee"
            },
            {
              "option_type": "action",
              "action_name": "Claws",
              "count": 1,
              "type": "melee"
            }
          ]
        },
        {
          "desc": "The chimera uses its fire breath in place of its bite, if the breath is available."
          "option_type": "multiple",
          "items": [
            {
              "option_type": "action",
              "action_name": "Fire Breath",
              "count": 1,
              "type": "ability"
            },
            {
              "option_type": "action",
              "action_name": "Horns",
              "count": 1,
              "type": "melee"
            },
            {
              "option_type": "action",
              "action_name": "Claws",
              "count": 1,
              "type": "melee"
            }
          ]
        },
        {
          "desc": "The chimera uses its fire breath in place of its horns, if the breath is available."
          "option_type": "multiple",
          "items": [
            {
              "option_type": "action",
              "action_name": "Bite",
              "count": 1,
              "type": "melee"
            },
            {
              "option_type": "action",
              "action_name": "Fire Breath",
              "count": 1,
              "type": "ability"
            },
            {
              "option_type": "action",
              "action_name": "Claws",
              "count": 1,
              "type": "melee"
            }
          ]
        }
      ]
    }
  }
}
```

The wording of the `desc`s is up for discussion, but this way we're using the existing and more widespread `desc` attribute to clarify things, without spamming `notes`.

Reply:
I went with omitting the notes in favor of using more specific descriptions. The only exception was the Tarrasque, where the description didn't list that it can use swallow only when it's grappling someone, so I left the description in there.